services/search/pkg/search/service.go

```diff
@@ -229,7 +249,7 @@ func (s *Service) searchIndex(ctx context.Context, req *searchsvc.SearchRequest,
 		rootName         string
 		permissions      *provider.ResourcePermissions
 	)
-	mountpointPrefix := ""
+	mountpointPrefix := req.GetRef().GetPath()
```

Comment:
This is gonna be replaced in case of "grant" spaces (shares) later on, no? So scoping a share wouldn't work?
I think you may have to prefix the scope-prefix with the grant-prefix to make that use case work, if that makes sense..